user-guide/Advanced_Functionality/Databases/Indexing_database/Profile_migration_to_elastic.md

```diff
@@ -13,7 +13,7 @@ When you [install an indexing database](xref:Installing_Elasticsearch_via_DataMi
 
 > [!NOTE]
 >
-> - XML storage is no longer supported from DataMiner 10.4.0 onwards. Upgrading to 10.4.0 will not be possible if the profile data is still stored in XML.
+> - XML storage is no longer supported from DataMiner 10.4.0 onwards. Upgrading to 10.4.0 will not be possible if the profile data is still stored in XML. Neither will it be possible to switch storage type in the SLNetClientTest tool.
```

Comment:
Do you mean that once you have upgraded to 10.4.0, it will no longer be possible to switch storage type?

Reply:
Yes correct, the buttons will no longer be in the migration window in client test tool